website/docs/d/batch_pool.html.markdown

```diff
@@ -92,3 +94,19 @@ A `auto_user` block exports the following:
 * `elevation_level` - The elevation level of the user identity under which the start task runs.
 
 * `scope` - The scope of the user identity under which the start task runs.
+
+---
+
+A `resource_file` block exports the following:
+
+* `auto_storage_container_name` - The storage container name in the auto storage account.
+
+* `blob_prefix` - The blob prefix used when downloading blobs from an Azure Storage container. Only the blobs whose names begin with the specified prefix will be downloaded. If a prefix is not specified, all the files in the container will be downloaded.
+
+* `file_Mode` - The file permission mode attribute in octal format. This property applies only to files being downloaded to Linux compute nodes. It will be ignored if it is specified for a `resource_file` which will be downloaded to a Windows node. If this property is not specified for a Linux node, then a default value of 0770 is applied to the file.
+
+* `file_path` - The location on the compute node to which to download the file, relative to the task's working directory. If the `http_url` property is specified, the `file_path` is required and describes the path which the file will be downloaded to, including the filename. Otherwise, if the `auto_storage_container_name` or `storage_container_url` property is specified, `file_path` is optional and is the directory to download the files to. In the case where `file_path` is used as a directory, any directory structure already associated with the input data will be retained in full and appended to the specified filePath directory. The specified relative path cannot break out of the task's working directory (for example by using '..').
```

Comment:
```suggestion
* `file_path` - The location on the compute node to which to download the file, relative to the task's working directory. If the `http_url` property is specified, the `file_path` is required and describes the path which the file will be downloaded to, including the filename. Otherwise, if the `auto_storage_container_name` or `storage_container_url` property is specified. 
```